iquip/apps/builder.py

```diff
@@ -290,6 +290,30 @@ def value(self) -> Optional[str]:
             return self.dateTimeEdit.dateTime().toString(Qt.ISODate)
         return None
 
+# TODO(AIJUH): Add feature for argInfo processing and other Scan type classes.
```

Comment:
Please write "Scan" in lower case, too!
And add one more empty line above the TODO comment.